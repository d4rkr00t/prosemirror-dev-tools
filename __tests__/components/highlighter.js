import React from "react";
import renderer from "react-test-renderer";

test.only("Item Highlight when child present", () => {
  const module = require("../../src/components/highlighter");
  var highligthed = module.Highlighter({ children: "1" });
  let tree = renderer.create(highligthed).toJSON();
  expect(tree).toMatchSnapshot();
});
