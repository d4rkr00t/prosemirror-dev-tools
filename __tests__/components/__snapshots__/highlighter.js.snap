// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Item Highlight when child present 1`] = `
<pre
  className="sc-bdVaJa gRLIpl"
  style={
    Object {
      "background": "transparent",
      "color": "white",
      "display": "block",
      "overflowX": "auto",
      "padding": "0.5em",
    }
  }
>
  <code>
    1
  </code>
</pre>
`;
